ArduCopter/mode_follow.cpp

```diff
@@ -100,22 +100,22 @@ void ModeFollow::run()
                 Vector3f accel_ned_mss;  // accel of lead vehicle
                 if (g2.follow.get_target_pos_vel_accel_NED_m(pos_ned_m, vel_ned_ms, accel_ned_mss))
                 if (pos_ned_m.xy().length_squared() > 1.0) {
-                    yaw_cd = get_bearing_cd(Vector2f{}, (pos_ned_m.xy() - pos_control->get_pos_target_NEU_cm().xy()).tofloat());
+                    yaw_rad = (pos_ned_m.xy() - pos_control->get_pos_target_NEU_cm().xy()).tofloat().angle();
```

Comment:
similar to the comment below I think we should double check this calc to confirm there is no change in behaviour